src/main/java/com/switchfully/parkshark_2024_10/user/Director.java

```diff
@@ -11,8 +11,9 @@ public class Director extends Person {
 
 
 
-       @OneToOne
-       @JoinColumn(name="FK_DIVISION_ID")
+//       @OneToOne
+//       @JoinColumn(name="FK_DIVISION_ID")
+    @Transient
        private Division division;
```

Comment:
I don't think this is correct, right? This removes the Director to Division relation which we don't want to do.

Reply:
No, but the division table does not exist yet, so this is a temporary measure so the user can be persisted in the test environment